cogs/Birthdays.py

```diff
@@ -117,17 +131,36 @@ async def get(
         if birthday is None:
             return await errorEmb(inter, "This user doesn't have a birthday set")
 
-        next_birthday = datetime.datetime.strptime(
-            birthday.birthday + f"/{dt.datetime.now().year}", "%d/%m/%Y"
-        )
-        if next_birthday < datetime.datetime.now():
-            next_birthday = datetime.datetime.strptime(
-                birthday.birthday + f"/{dt.datetime.now().year + 1}", "%d/%m/%Y"
-            )
+        days, discord_date = await get_days_until_birthday(birthday)
+        await QuickEmb(
+            inter,
+            f"{user.mention}'s birthday is in {days} Days." f"{discord_date}",
+        ).send()
+
+    @birthday.sub_command(name="next", description="Get the next birthday")
+    async def next(self, inter: disnake.ApplicationCommandInteraction):
+        x = 1000
+        next_birthday = None
+        member = None
+        # loop gets next birthday
```

Comment:
damn this is inefficient... PLEASE OPTIMIZE @LevaniVashadze (git blame says this is u )

Reply:
ok i'll try

Reply:
fixed